ci_framework/hooks/playbooks/ceph-deploy.yml

```diff
@@ -0,0 +1,23 @@
+# Following those notes: https://github.com/fultonj/zed/tree/main/edpm
+- hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Load all of our cifmw variables
+      ansible.builtin.include_vars:
+        file: "{{ cifmw_basedir }}/artifacts/custom-params.yml"
+    - name: Run make_ceph
+      vars:
+        make_ceph_environment:
+          TIMEOUT: 90
```

Comment:
Do we have plans to pass additional parameters in case we want to support network isolation? We have the ability to attach the `multus` network to the `Ceph` pod and start it on a specific `MON_IP` [1].
It would be interesting adding this part, maybe in a follow up PR, not sure if @stuggi  has an example other than what's described in [1].
Basically, when the ctlplane is deployed w/ network-isolation, we might want to attach the `storage` network to the ceph pod and deploying it on the IP address reserved on that `NAD`.

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/master/scripts/gen-ceph-kustomize.sh#L222

Reply:
hmmm ok, so we may pass that parameter differently. Lemme nudge an update, you'll see how cool the framework and its hooks may be :). Thanks for that pointer!